src/QuadForm/Herm/Genus.jl

```diff
@@ -1556,14 +1556,17 @@ function representative(G::HermGenus)
   if !is_integral(G)
     s = denominator(_scale(G))
     L = representative(rescale(G, s))
-    return rescale(L, 1//s)
+    L = rescale(L, 1//s)
+    return L
```

Comment:
Why?

Reply:
In my first changes, I was caching the representative on the genus symbol so I changed like this. They I left this idea aside for later, especially because my example would not fit. I forgot to turn it back as it was.